python/desc/imsim/imSim.py

```diff
@@ -1,39 +1,43 @@
-"""
-This is the imSim program, used to drive GalSim to simulate the LSST.  Written
-for the DESC collaboration and LSST project.  This version of the program can
-read phoSim instance files as is. It leverages the LSST Sims GalSim interface
-code found in sims_GalSimInterface.
-"""
-import os
 import sys
-import logging
-import argparse
+from collections import namedtuple
```

Comment:
Since this file now just contains the parser should it have Parser type name (instead if imSim.py) instead?  Or is the intention to put a lot of code in here?